platforms-whitepaper/v1alpha1/paper.md

```diff
@@ -101,6 +101,11 @@ are arranged as a compelling internal product. Autonomous delivery teams can
 make use of the platform to deliver product features at a higher pace, with 
 reduced coordination."
 
+The goal of any platform should be to drive organizational efficiency and 
```

Comment:
IMO we shouldn't insist on organizational efficiency as the goal of a platform here. Some orgs I work with are concerned more about cost savings and security than productivity for example.

The first sentence of the next paragraph briefly highlights the idea that a platform should be driven by user requirements like productivity or others. Instead of adding a paragraph could we could update that first sentence of the next one for greater clarity as follows? Thanks!

"The specific set of capabilities and scenarios supported by a platform should be determined by the needs of users."